src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperRunner.cs

```diff
@@ -19,28 +18,16 @@ public class TagHelperRunner
         /// <paramref name="context"/>'s <see cref="ITagHelper"/>s.</returns>
         public async Task<TagHelperOutput> RunAsync([NotNull] TagHelperExecutionContext context)
         {
-            return await RunAsyncCore(context, string.Empty);
+            return await RunAsyncCore(context);
```

Comment:
Do you need `RunAsynCore` at this point since there's only one consumer (this method)? Consider merging it.